### ISSUE TITLE
Icons in information banners are not aligned to the text

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-frontend#10464]
### Describe the problem and steps to reproduce it:
1. Log in to AMO with an account that has some collections
2. Open an add-on detail page
3. Scroll down to the Add to collection card
4. Add the add-on to one of your collections
5. Notice the confirmation banner displayed

### What happened?
The icon and the message in the banner don't have the same alignment

### What did you expect to happen?
The icon and the message have the same alignment 

### Anything else we should know?
- this reproduces on all AMO servers
- this is most likely the case with all the information banners displayed on the site, so this is just one way to reproduce the issue

 
![image](https://user-images.githubusercontent.com/31961530/115570112-8693f580-a2c6-11eb-956f-bfea7af2f586.png)
![image](https://user-images.githubusercontent.com/31961530/115570133-8a277c80-a2c6-11eb-96b9-d6d5a54bcc03.png)
![image](https://user-images.githubusercontent.com/31961530/115570151-8dbb0380-a2c6-11eb-83b5-943a4bde9cc6.png)

**Comment by bobsilverberg** [comment]
I feel like this exact same issue came up, and was fixed, awhile ago. Does that ring a bell for you too @AlexandraMoga?

**Comment by AlexandraMoga** [comment]
@bobsilverberg Yes, I'm sure this was brought up during the font size increase and was fixed with a bundle of other issues. Now it resurfaced.

**Comment by AlexandraMoga** [comment]
Actually, comparing to the screenshots from above, AMO-prod is looking better (items still look slightly misaligned to me), so the problem reported now is only in -dev and stage:

[Prod]
![image](https://user-images.githubusercontent.com/31961530/115574543-65cd9f00-a2ca-11eb-8943-627c1df0aa35.png)

**Comment by bobsilverberg** [comment]
It looks like it relates to the `line-height` change from https://github.com/mozilla/addons-frontend/commit/ede59eb38ed31cb5736b536cf1469c616a9f0518. I'm not sure if we want a specific fix to target these notifications with icons, or if this is something we should address via https://github.com/mozilla/addons-frontend/issues/10251 when we take a fresh look at all line-heights.

WDYT @willdurand? I don't think it's something that necessarily needs to be fixed immediately.

**Comment by AlexandraMoga** [comment]
This issue is verified fixed on stage:

![image](https://user-images.githubusercontent.com/31961530/117945099-a727fc00-b316-11eb-94bc-f73e9c6de0a2.png)
![image](https://user-images.githubusercontent.com/31961530/117945116-abecb000-b316-11eb-8a05-2d7e764c39f3.png)

**Comment by willdurand** [comment]
@bobsilverberg looking at the screenshots above, it still looks like we could add some more margin to the icons, they are not exactly centered.

**Comment by bobsilverberg** [comment]
> @bobsilverberg looking at the screenshots above, it still looks like we could add some more margin to the icons, they are not exactly centered. WDYT?

I agree, it can use a few more pixels at the top. I'll file and issue.
